Cargo.toml

```diff
@@ -122,6 +121,10 @@ kvproto = { git = "https://github.com/pingcap/kvproto.git" }
 log_wrappers = { path = "components/log_wrappers" }
 engine = { path = "components/engine" }
 
+[dependencies.grpcio]
```

Comment:
why not changing above uuid at the same time?

Reply:
Can you rephrase the question? I don't understand.

Reply:
Oh, do you mean to write the uuid dependency the way the grpcio dependency is written?

I can do that, yes.

I shouldn't have reformatted grpcio though. It was an accident.